modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptInversifyClientCodegen.java

```diff
@@ -52,7 +62,8 @@ public TypeScriptInversifyClientCodegen() {
         apiPackage = "api";
         modelPackage = "model";
 
-        this.reservedWords.add("map");
+        // TODO Are those case sensitive or insensitive
+        super.registerReservedWordsCaseInsensitive(Collections.singletonList("map"));
```

Comment:
case sensitive

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)